src/Datapoint.c

```diff
@@ -269,7 +272,7 @@ ds_datapoint_t *dpoint_from_string(char *str, int len)
 {
   ds_datapoint_t *d = NULL;
   char *varname;
-  int varlen;
+  int name_length;
```

Comment:
I'm being picky.  Since we're reading from the middle of a string, we don't expect to see a null terminator nearby.  So this is length is not `varlen` in the sense of `strlen` + 1.  I just renamed to avoid confusion.  I hope.